repl.js

```diff
@@ -291,6 +294,30 @@ class CombeeQueue {
 
     return matches;
   }
+
+  /**
+   * Dump jobs, based on a filter, to a file.
+   *
+   * @param      {String}   jobType      The type of job (i.e. 'active', 'waiting', etc).
+   * @param      {Object}   [filter={}]  The sift-compatible filter
+   */
+  async dump(jobType, filter = {}) {
+    const matches = await this._find(jobType, filter);
```

Comment:
I think it'd make sense to add a `forEach` method so that we could write the jobs out without needing to have them all in memory.

Reply:
Ah yeah, makes sense

Reply:
Agreed, there could be a ton of jobs here - so paginating would be ideal. It might make sense to add a batch processor to handle matches as we [find them here](https://github.com/ttacon/combee/blob/master/repl.js#L288).

Reply:
On that - https://github.com/ttacon/combee/pull/14